bucket/pychess.json

```diff
@@ -0,0 +1,22 @@
+{
+    "homepage": "http://pychess.org/",
+    "description": "A chess client for Linux and Windows",
+    "license": "GPL-3.0",
+    "version": "0.99.4",
+    "url": "https://github.com/pychess/pychess/releases/download/0.99.4/pychess-0.99.4-win32.msi",
+    "hash": "aa4464cceb06686d657d25d7de2e14f2de84b6f3d7511c1bcd45dbcfa570e55d",
+    "bin": "pychess.exe",
+    "shortcuts": [
+        [
+            "pychess.exe",
+            "PyChess"
+        ]
+    ],
+    "checkver": {
```

Comment:
I actually tried using GitHub regex but for some reason I kept receiving an error:
```
pychess: couldn't find new version on https://github.com/pychess/pychess/
```
… so I had to come up with this regex instead.

Reply:
![Proof](https://i.imgur.com/utf2iiP.png)

Reply:
hmmmm.... It seems to be working perfectly now. Maybe because I'm using
```json
"checkver" {
    "github": "https://github.com/pychess/pychess"
}
```
instead of 
```json
"checkver" {
    "url": "https://github.com/pychess/pychess",
    "re": "github"
}
```